<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="valueInput"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="enqueueButton">
        <property name="text">
         <string>Enqueue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dequeueButton">
        <property name="text">
         <string>Dequeue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="splitButton">
        <property name="text">
         <string>Split</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="mainQueueList"/>
      </item>
      <item>
       <widget class="QListWidget" name="positiveQueueList"/>
      </item>
      <item>
       <widget class="QListWidget" name="negativeQueueList"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
